Frontend/src/admin/techs-tab/TechsTab.tsx

```diff
@@ -15,16 +16,32 @@ import {
 } from '@material-ui/core';
 import ListItemIconSansPadding from '../../common/components/ListItemIconSansPadding';
 import PersonIcon from '@material-ui/icons/Person';
+import AddTechForm from './AddTechForm';
+import { addTech } from '../../common/api/endpoints';
 
 const TechsTab = () => {
-  const [refreshState] = useRefresh();
+  const [refreshState, refresh] = useRefresh();
   const getTechs = mockedServices.getTechs;
+  const removeTech = useServices().removeTech;
   const data = usePromise(() => Promise.all([getTechs()]), [refreshState]);
```

Comment:
Ummm, ok. Should I also change all the other `useService()`s?
![image](https://user-images.githubusercontent.com/24509819/120024238-28f27780-bfef-11eb-8274-f73225be98a4.png)

Reply:
Here it is sensible, we want to merge two requests. 
Buuuut, maybe it would be nice to move this merging logic to services and make one service with Promise.all instead of using Promise.all here